manuscript/manuscript.Rmd

```diff
@@ -467,6 +460,13 @@ But classical approaches can be difficult to generalize when the underlying mode
 In contrast, the process of training an RL algorithm on a more complex problem is no different than training on a simple one: we only need access to a simulation which can generate plausible future states in response to possible actions.
 This flexibility of RL could allow us to attain better decision-making insight for our most realistic ecological models such as the individual-based models used in the management of forests and wildfire [@sortie; @wildfire], disease [@covid], marine ecosystems [@ecopath], or global climate change [@dice].
 
+<!-- Model free is great bc we don't know the true models -->
+The rapidly expanding class of model-free RL algorithms is particularly appealing given the ubiquitous presence of model uncertainty in ecological dynamics.
+Rarely do we know the underlying functional forms for ecological processes.
+Methods which must first assume something about the structure or functional form of a process before estimating the corresponding parameter can only ever be as good as those structural assumptions.
+Frequently available ecological data is insufficient to distinguish between possible alternative models [@knape_are_2012], or the correct model may be non-identifiable with any amount of data.
+Model-free RL approaches offer a powerful solution for this thorny issue.
```

Comment:
I wonder if we can give some more intuition here.  I think ecological readers are tempted to think of this as meaning that the neural network is complex enough to model the process even when our analytical equations are not.

but really I think the point comes down to something about a policy function which gives the optimal action given the state(s), a = g(x_t ... x_1),  is often easier to estimate than the transition function.  Winning a chess game isn't about who is best at predicting what the the other player is going to do, it's just about knowing how to react to whatever they do.  but yeah, maybe that'll just get lost anyway so this is fine as is.